jenkins-scripts/dsl/_configs_/OSRFUNIXBase.groovy

```diff
@@ -38,6 +38,59 @@ class OSRFUNIXBase extends OSRFBase
              git clone https://github.com/gazebo-tooling/release-tools scripts -b \$RTOOLS_BRANCH
              """.stripIndent())
       }
+      publishers {
+        // Manual insertion of xml for Naginator plugin because of this issue https://issues.jenkins.io/browse/JENKINS-66458
+        configure { project -> 
```

Comment:
We probably want to move the retry/naginator configure block after the `postBuildScripts` so the remove of the labels in the node happens before we launch the build back.

Reply:
Will run a test job once the debbuilders finish running. The logic is that since that shutdown is on a step block, whenever we don't find the regex the previous step fails and therefore the shutdown is not run.